### ISSUE TITLE
[Word]: NVDA is not announcing type of change in the document when navigating with change button.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8287]
###Steps to reproduce:
1. Open a word doc 
2. Turn ON track Changes
3. Make some Changes in the document
4. Now Navigate to Ribbons Tab --> Review-->Changes (Next/Previous)
5. Observe that NVDA is just announcing the next changed content that has been selected but there is no info about the type of change (Like : Formatted- Bold Italic, Deleted, Inserted etc)
 
### Expected behavior:
NVDA should announce the type of change as well as change content as announced by narrator.

### Actual behavior:
NVDA is not announcing type of change in the document when navigating with change button. It is announcing everything properly if we are navigating with Arrow keys in the document.

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
Actually, you can easily jump from a change to another in browse mode by pressing a.
This is much more efficient than going to the menu and pressing the button everytime.
However, I will let the decision to the community or NV Access if they want to implement the reporting of the coresponding change when navigating to it via the ribbon menu instead of using NVDA's browse mode.
